ocw/tests/test_metrics.py

```diff
@@ -208,41 +229,6 @@ def test_function_run(self):
         expected_result.fill(-300)
         np.testing.assert_array_equal(self.mean_bias.run(self.target_dataset,self.reference_dataset), expected_result)
 
-    def test_function_run_abs(self):
-        '''Test mean bias function between reference dataset and target dataset with abs as True.'''
-        expected_result = np.zeros((5, 5), dtype=np.int)
-        expected_result.fill(300)
-        np.testing.assert_array_equal(self.mean_bias.run(self.reference_dataset, self.target_dataset, True), expected_result)
-
-
-class TestSpatialMeanOfTemporalMeanBias(unittest.TestCase):
```

Comment:
@MJJoyce, I agree with the point. However, I just could not leave unreasonable functions while updating metrics and test_metrics. It is too hard to manage many branches to separate all trivial updates while working to an approaching deadline. I never expect that anyone is using this function for any purpose. The ticket's title can be misleading. How do you think about revising the JIRA issue?